### PR TITLE
Added Alumni Directory & Bug Tracker

### PR DESCRIPTION
**Pull request by ialkamal**
## Added Alumni Directory & Bug Tracker

**Description:** 
Added the following features:
[1] Alumni Directory
[2] Bug Tracker

  
  
  
  

### Checklist

 * [ ] My code follows the style guidelines of this project
 * [ ] I have performed a self-review of my own code
 * [ ] My code has been reviewed by at least one peer
 * [ ] I have commented my code, particularly in hard-to-understand areas
 * [ ] I have made corresponding changes to the documentation
 * [ ] My changes generate no new warnings
 * [ ] I have added tests that prove my fix is effective or that my feature works
 * [ ] New and existing unit tests pass locally with my changes
 * [ ] There are no merge conflicts

### CODE REVIEW

File: src/components/authentication/Success.js [hunk 1]
```diff
@@ -10,6 +10,7 @@ const Success = (props) => {
 
   // save token to localStorage
   useEffect(() => {
+    console.log('REACHED');
```
Comment (Plofland) [inline src/components/authentication/Success.js]: We can get rid of this console.log unless you still need it.

File: src/assets/Directory.js [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+        <Content>
+          <List
+            grid={{ gutter: 16, column: 2 }}
+            dataSource={props.users}
+            pagination={{
+              onChange: (page) => {},
+              pageSize: 6,
+            }}
+            renderItem={(user) => (
+              //   props.user.id !== user.id && (
```
Comment (Plofland) [inline src/assets/Directory.js]: We can get rid of this since you moved it to line 58.

File: src/assets/Directory.js [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+                      </span>
+                      <span>
+                        {user.github_username ? (
+                          <a
+                            href={`https://github.com/${user.github_username}`}
+                          >
+                            <b>Github</b>
+                          </a>
+                        ) : (
+                          <b>Github</b>
```
Comment (Plofland) [inline src/assets/Directory.js:121]: Maybe we can change this to just be blank if there is no GitHub username provided by the user

**Review by Plofland — APPROVED** [review]
Great job, can't wait to see what this looks like in the app.

File: src/assets/Directory.js (hunk 2 above)
Comment (Plofland) [reply src/assets/Directory.js]: I'll probably make this change before you when I go into the file to move over the styles

File: src/assets/Directory.js (hunk 3 above)
Comment (Plofland) [reply src/assets/Directory.js:121]: I'll probably make this change before you when I go into the file to move over the styles

Comment (Plofland) [reply src/assets/Directory.js:121]: I'll change it back if it doesn't look right

File: src/components/authentication/Success.js (hunk 1 above)
Comment (ialkamal) [reply src/components/authentication/Success.js]: removed

File: src/assets/Directory.js (hunk 2 above)
Comment (ialkamal) [reply src/assets/Directory.js]: done

File: src/assets/Directory.js (hunk 3 above)
Comment (ialkamal) [reply src/assets/Directory.js:121]: You can modify this as you think about styling the component. Currently this is how it looks, black if no link and blue if it's linked. 
image

File: src/components/authentication/Success.js (hunk 1 above)
Comment (Plofland) [reply src/components/authentication/Success.js]: Great

File: src/assets/Directory.js (hunk 2 above)
Comment (Plofland) [reply src/assets/Directory.js]: Thank you

File: src/assets/Directory.js (hunk 3 above)
Comment (Plofland) [reply src/assets/Directory.js:121]: Thanks for the screenshot, it really helps to visualize the code.
Ah I see, I'll think about the styling of this directory this week.

**Review by robo-chick — APPROVED** [review]
Nice work!

**Pull request merged**